wrappers/react-wrapper/src/helpers.tsx

```diff
@@ -0,0 +1,231 @@
+    'HotColumn components do not support any children.';
+
+/**
+ * Message for the warning thrown if the Handsontable instance has been destroyed.
+ */
+export const HOT_DESTROYED_WARNING = 'The Handsontable instance bound to this component was destroyed and cannot be' +
+  ' used properly.';
+
+/**
+ * Default classname given to the wrapper container.
+ */
+export const DEFAULT_CLASSNAME = 'hot-wrapper-editor-container';
+
+/**
+ * Logs warn to the console if the `console` object is exposed.
+ *
+ * @param {...*} args Values which will be logged.
+ */
+export function warn(...args: any[]) {
+  if (typeof console !== 'undefined') {
+    console.warn(...args);
+  }
+}
+
+/**
+ * Detect if `hot-renderer` or `hot-editor` is defined, and if so, throw an incompatibility warning.
+ *
+ * @returns {boolean} 'true' if the warning was issued
+ */
+export function displayObsoleteRenderersEditorsWarning(children: ReactNode): boolean {
+  if (hasChildElementOfType(children, 'hot-renderer')) {
+    warn(OBSOLETE_HOTRENDERER_WARNING);
+    return true;
+  }
+  if (hasChildElementOfType(children, 'hot-editor')) {
+    warn(OBSOLETE_HOTEDITOR_WARNING);
+    return true;
+  }
+
+  return false
+}
+
+/**
+ * Detect if children of specified type are defined, and if so, throw an incompatibility warning.
+ *
+ * @param {ReactNode} children Component children nodes
+ * @param {ComponentType} Component Component type to check
+ * @returns {boolean} 'true' if the warning was issued
+ */
+export function displayChildrenOfTypeWarning(children: ReactNode, Component: ComponentType): boolean {
+  const childrenArray: ReactNode[] = React.Children.toArray(children);
+
+  if (childrenArray.some((child) => (child as React.ReactElement).type !== Component)) {
+    warn(UNEXPECTED_HOTTABLE_CHILDREN_WARNING);
+    return true;
+  }
+
+  return false
+}
+
+/**
+ * Detect if children is defined, and if so, throw an incompatibility warning.
+ *
+ * @param {ReactNode} children Component children nodes
+ * @returns {boolean} 'true' if the warning was issued
+ */
+export function displayAnyChildrenWarning(children: ReactNode): boolean {
+  const childrenArray: ReactNode[] = React.Children.toArray(children);
+
+  if (childrenArray.length) {
+    warn(UNEXPECTED_HOTCOLUMN_CHILDREN_WARNING);
+    return true;
+  }
+
+  return false
+}
+
+/**
+ * Check the existence of elements of the provided `type` from the `HotColumn` component's children.
+ *
+ * @param {ReactNode} children HotTable children array.
+ * @param {String} type Either `'hot-renderer'` or `'hot-editor'`.
+ * @returns {boolean} `true` if the child of that type was found, `false` otherwise.
+ */
+function hasChildElementOfType(children: ReactNode, type: 'hot-renderer' | 'hot-editor'): boolean {
+  const childrenArray: ReactNode[] = React.Children.toArray(children);
+
+  return childrenArray.some((child) => {
+      return (child as React.ReactElement).props[type] !== void 0;
+  });
```

Comment:
Can I ask to check if the linter is configured correctly? There should be 2 spaces as indentation, not 4. The `void 0` I would replace with `undefined`